### ISSUE TITLE
How to specify the fonts for Export to pdf

### ISSUE THREAD

**Issue by wtpchan** [xpert-framework/xpert-framework#16]
We make use of the CRUD function by the maker to produce a pdf report.   There are some Chinese fields which use a specific font.   The current report produced by maker will generate a null field for Chinese contents.  How to set the font for the particular field ?

**Comment by ayslanms** [comment]
Do you have problem only with PDF?

**Comment by wtpchan** [comment]
Yes, export to Excel and CSV shows the Chinese character while PDF does not.   The PDF needs to use or embed the Chinese font in order to show correctly.

I have added some Chinese data to the showcase for your reference:

http://showcase.xpertsistemas.com.br/views/crud/country/listCountry.jsf

**Comment by ayslanms** [comment]
This is a primefaces `p:dataExporter` component related problem
